sites/fast-website/src/public/index.html

```diff
@@ -162,23 +168,33 @@ <h4>Other</h4>
 
         <!-- community:start -->
         <section class="section">
-            <site-content-placement-container section="community"></site-content-placement-container>
+          <site-content-placement-container section="community"></site-content-placement-container>
         </section>
         <!-- community:end -->
 
         <!-- footer:start -->
           <fast-divider class="divider"></fast-divider>
           <div class="footer">
-            <site-navigation>
-              <fast-anchor class="footer-navigation-item" appearance="lightweight" href="#">License</fast-anchor>
-              <fast-anchor class="footer-navigation-item" appearance="lightweight" href="#">Privacy & Cookies</fast-anchor>
-              <fast-anchor class="footer-navigation-item" appearance="lightweight" href="#">Terms of Use</fast-anchor>
-              <fast-anchor class="footer-navigation-item" appearance="lightweight" href="#">&copy; 2020 Microsoft</fast-anchor>
-              <fast-anchor slot="end" appearance="lightweight" href="#">FAST</fast-anchor>
+            <site-navigation class="footer-navigation">
+              <ul>
+                <li><fast-anchor appearance="lightweight" href="#">License</fast-anchor></li>
+                <li><fast-anchor appearance="lightweight" href="#">Privacy & Cookies</fast-anchor></li>
+                <li><fast-anchor appearance="lightweight" href="#">Terms of Use</fast-anchor></li>
+                <li><fast-anchor appearance="lightweight" href="#">&copy; 2020 Microsoft</fast-anchor></li>
+              </ul>
             </site-navigation>
+            <fast-anchor class="footer-brand" appearance="lightweight" href="#">
+              <%= require("svg/icon-brand.svg") %>
+            </fast-anchor>
           </div>
         <!-- footer:end -->
       </div>
     </fast-design-system-provider>
+    <script>
```

Comment:
We should probably make the hamburger button its own fast element-based component to move this function into a module.

Reply:
the button is moved into the component